quarkus/admin/build.gradle.kts

```diff
@@ -87,8 +87,8 @@ distributions {
     contents {
       from(runScript)
       from(project.layout.buildDirectory.dir("quarkus-app"))
-      from("../../NOTICE")
-      from("../../LICENSE-BINARY-DIST").rename("LICENSE-BINARY-DIST", "LICENSE")
+      from("distribution/NOTICE")
+      from("distribution/LICENSE")
```

Comment:
Is it possible to leverage on a super set of LICENSE and NOTICE? So that we don't have to place them in each module.

Reply:
The `LICENSE` and `NOTICE` in the root folder are for the source distribution.

admin and server distributions are different `LICENSE` and `NOTICE`: you can see the content is different. So, each distribution has its own specific `LICENSE` and `NOTICE`.

Reply:
They are different. I am wondering if it's possible to provide a super set so that, we don't have to to go to each individual binary module to generate a different LICENSE and NOTICE, which adds a lot of burden for releasing. IIUC, we potentially have to check each individual module as their jar files are different, https://github.com/apache/polaris/pull/1258#issuecomment-2760810813.

Reply:
Ah, you mean a "common" share `LICENSE` and `NOTICE` (the common denominator), and add the specific for each ?

Reply:
yeah, that's a close one. I was thinking about an option that we just got a super set of license and notice so that, all other can just reuse it.

 Another option is having an automation tool, not sure if that's possible, and how much effort we need to build such a tool.